showdependencygraph.cgi

```diff
@@ -76,16 +76,16 @@ my $add_link = sub {
 
       $link_text .= $dependson;
       $link_text .= ($dependson_status eq 'RESOLVED') ? '[' : '([';
-      $link_text .= "$dependson";
+      $link_text .= '"' . $dependson;
```

Comment:
This seems like it's likely to break in the future... could you build up the variable to be quoted in a separate variable and then just `$link_text .= '"$dependson_text"';` instead?